dm/ctl/master/config.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"github.com/pingcap/dm/dm/pb"
+
+	"github.com/spf13/cobra"
+)
+
+// NewConfigCmd creates a OperateSchema command.
+func NewConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config <command>",
+		Short: "manage config operations",
+	}
+	cmd.AddCommand(
+		newConfigTaskCmd(),
+		newConfigSourceCmd(),
+		newConfigMasterCmd(),
+		newConfigWorkerCmd(),
+	)
+	cmd.PersistentFlags().StringP("output", "o", "", "write config to file")
+	return cmd
+}
+
+func newConfigTaskCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "task [task-name]",
+		Short: "manage or show task configs",
+		RunE:  configTaskList,
```

Comment:
Maybe use anonymous functions in the same format as above?

Reply:
which format?

Reply:
here? 

https://github.com/pingcap/dm/pull/1746/files/4380557bc6f5ccd932934bd92325d6b6ea6be9ef#diff-fd7922b00c9e44d86f571af7fd5bd503e038c8576214ae84fab6970c5684ed49R78-R84

use `func(cmd *cobra.Command, args []string) error` instead of `configTaskList`